File: Examples/Java/SDK/src/main/java/com/aspose/cells/cloud/examples/worksheet/DeleteWatermarkBackground.java
package com.aspose.cells.cloud.examples.worksheet;

import com.aspose.cells.api.CellsApi;
import com.aspose.cells.cloud.examples.Configuration;
import com.aspose.cells.cloud.examples.Utils;
import com.aspose.storage.api.StorageApi;

import java.io.IOException;
import java.nio.file.Files;
import java.nio.file.Path;
import java.nio.file.StandardCopyOption;

public class DeleteWatermarkBackground {

    public static void main(String... args) throws IOException {
        // ExStart: delete-watermark-background
        try {
            // Instantiate Aspose Storage API SDK
            StorageApi storageApi = new StorageApi(Configuration.apiKey, Configuration.appSID, true);

            // Instantiate Aspose Words API SDK
            CellsApi cellsApi = new CellsApi(Configuration.apiKey, Configuration.appSID, true);
            Path inputFile = Utils.getPath(DeleteWatermarkBackground.class, "Sample1.xlsx");
            Path outputFile = Utils.getPath(DeleteWatermarkBackground.class, "Sample2.xlsx");

            storageApi.PutCreate(inputFile.getFileName().toString(), null, Utils.STORAGE, inputFile.toFile());

            String sheetName = "Sheet1";

            cellsApi.DeleteWorkSheetBackground(inputFile.getFileName().toString(), sheetName, Utils.FOLDER,
                    Utils.STORAGE);

            com.aspose.storage.model.ResponseMessage sr = storageApi.GetDownload(inputFile.getFileName().toString(),
                    null, Utils.STORAGE);

            Files.copy(sr.getInputStream(), outputFile, StandardCopyOption.REPLACE_EXISTING);
        }

        catch (Exception e) {
            e.printStackTrace();
        }
        // ExEnd: delete-watermark-background

    }
}

File: Examples/DotNet/CSharp/Worksheet/DeleteWatermarkBackground.cs
using System;
using Com.Aspose.Cells.Api;
using Com.Aspose.Cells.Model;
using Com.Aspose.Storage.Api;

namespace Worksheet
{
    class DeleteWatermarkBackground
    {
        public static void Run()
        {
            // ExStart:1
            CellsApi cellsApi = new CellsApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            String fileName = "WorkSheetBackground_Sample_Test_Book.xls";
            String sheetName = "Sheet1";
            String folder = "";
            String storage = "";

            try
            {
                // Upload source file to aspose cloud storage
                storageApi.PutCreate(fileName, "", "", System.IO.File.ReadAllBytes(Common.GetDataDir() + fileName));

                // Invoke Aspose.Cells Cloud SDK API to delete background or watermark of worksheet
                SaaSposeResponse apiResponse = cellsApi.DeleteWorkSheetBackground(fileName, sheetName, folder, storage);

                if (apiResponse != null && apiResponse.Status.Equals("OK"))
                {
                    Console.WriteLine("Delete Background or Watermark of Excel Worksheet!");
                    Console.ReadKey();
                }
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}
